### DISCUSSION TITLE
Telling people what problem this project solves

### DISCUSSION THREAD

**Discussion by guy-roberts** [acacode/swagger-typescript-api#1013] [Q&A]
I just came across this project and am trying to see if it is a way to parse a swagger document and auto generate boiler plate code to support the API in a front end app.  
But I am not sure, because the README.md kind of assumes I already know what the purpose of the project is.

**Comment by rscott78** [comment]
Agreed! I was actually recommended this project by a different developer, so I knew what it did. However, the docs don't actually explain how to use the generated output, so we are sort-of left to stumble around to figure it out on our own.
